app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntrySheet.kt

```diff
@@ -376,6 +375,14 @@ class BloodSugarEntrySheet : BottomSheetActivity(), BloodSugarEntryUi, RemoveBlo
     bloodSugarDateButton.visibleOrGone(isVisible = true)
   }
 
+  override fun showBloodSugarUnitPreferenceMmol() {
```

Comment:
Do you think we should name these methods as `setBloodSugarUnitPreferenceMmol` or `setBloodSugarUnitPreferenceMg`, since we are not actually showing the button, but rather just setting the unit preference. WDYT?

Reply:
Yes this makes sense and we're also not setting but instead setting the label for it. So changed it to `setBloodSugarUnitPreferenceLabelToMmol` and `setBloodSugarUnitPreferenceLabelToMg`